content/en/methods/statuses.md

```diff
@@ -1482,18 +1482,18 @@ ID of the status in the database
 
 ```javascript
 {
-  "url": "https://www.youtube.com/watch?v=OMv_EPMED8Y",
-  "title": "♪ Brand New Friend (Christmas Song!)",
+  "url": "https://peertube.social/videos/watch/7a087e5f-68e0-4359-8035-bfd9752e9b2e",
+  "title": "What is Mastodon ?",
   "description": "",
   "type": "video",
-  "author_name": "YOGSCAST Lewis & Simon",
-  "author_url": "https://www.youtube.com/user/BlueXephos",
-  "provider_name": "YouTube",
-  "provider_url": "https://www.youtube.com/",
-  "html": "<iframe width=\"480\" height=\"270\" src=\"https://www.youtube.com/embed/OMv_EPMED8Y?feature=oembed\" frameborder=\"0\" allowfullscreen=\"\"></iframe>",
+  "author_name": "Opensource",
```

Comment:
is this a real example payload?

Reply:
I have fixed the syntax errors, apologies for them.

As the YouTube example is (by your standards) correct, then this is correct too. It does the exact same thing, but with Peertube.